ergo/distributions/histogram.py

```diff
@@ -194,14 +240,32 @@ def from_pairs(cls, pairs):
         logps = onp.log(onp.array(densities) / sum(densities))
         return cls(logps, scale_min=scale_min, scale_max=scale_max)
 
-    def to_pairs(self):
+    def to_pairs(self, total_p: float = 1.0):
```

Comment:
similarly to the changes to `normalize`, maybe this should live as its own `downscale_p` function outside of `to_pairs`

it could also just not even be part of `HistogramDist` and instead be something that's done to the pairs by Elicit

(I like that it's so incredibly easy to do this here, but would also be pretty easy to do in Elicit)